lib/rake/adhoc.rake

```diff
@@ -85,4 +85,28 @@ Note: Consumes AWS resources until `adhoc:stop` is called.'
       Rake::Task['adhoc:validate'].invoke
     end
   end
+
+  namespace :bugcrowd do
+    timed_task_with_logging environment: 'adhoc:environment' do
+      ENV['DATABASE'] = '1'
+      ENV['STACK_NAME'] = 'adhoc-bugcrowd'
+      ENV['CDN_ENABLED'] = '1'
+    end
```

Comment:
This did not work correctly, the stack creation was initiated with the name:  `adhoc-production`

Reply:
This doesn't work because the `adhoc:environment` code runs first, so setting the env variables here is too late. We could consider directly editing the `@cfn` object's `@template`, but that feels a little messier than I'd like. Will close this for now.